src/foam/u2/view/SpinnerView.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'SpinnerView',
```

Comment:
How is this different from using `.tag(foam.u2.LoadingSpinner)`

Reply:
i need to push just a single spinner to the stack

Reply:
Does `ctrl.stack.push({class: 'foam.u2.LoadingSpinner', size: <some size>})` not work?

Reply:
ah i was using it as a view in a stackblock before so didnt work, what you suggested seems to work, thanks!